ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -21,16 +21,28 @@ import Foundation
 public enum CocoaPodUtils {
   // MARK: - Public API
 
-  /// Information associated with an installed pod.
-  public struct PodInfo {
+  public struct VersionedPod {
     /// Public name of the pod.
-    var name: String
+    let name: String
 
     /// The version of the pod.
-    var version: String
+    let version: String
+  }
+
+  /// Information associated with an installed pod.
+  public struct PodInfo {
+    let versionedPod: VersionedPod
```

Comment:
This doesn't need to be changed at all but I think is a great time to mention a new feature from Swift 5.1 that would enable something cool.. you could essentially expose the properties of `VersionedPod` on `PodInfo` with autocomplete and everything with `@dynamicMemberLookup` (Swift 4.2) and `KeyPath`s (5.1).

```
@dynamicMemberLookup
public struct PodInfo {
  let versionedPod: VersionedPod
  
  subscript<T>(dynamicMember keyPath: KeyPath<VersionedPod, T>) -> T {
        return versionedPod[keyPath: keyPath]
    }
}
```

Would let you write `somePodInfo.name` and `somePodInfo.version` natively :) 

See the "Combining key paths with @dynamicMemberLookup" from this blog post for more info:
https://www.avanderlee.com/swift/dynamic-member-lookup/

I remember reading about it but couldn't think of many good use cases but this is a perfect one! Requires Swift 5.1 so we can't use it at the moment though :P